Orleans-Configuration-Guide/Server-Configuration.md

```diff
@@ -100,3 +100,127 @@ Primary is designated in the configuration file with the following setting withi
+        StatisticsPerfCountersWriteInterval = TimeSpan.FromMinutes(10),
+        StatisticsLogWriteInterval = TimeSpan.FromMinutes(10),
+        StatisticsWriteLogStatisticsToTable = false,
+        StatisticsCollectionLevel = StatisticsLevel.Info,
+
+        // Tracing
+        DefaultTraceLevel = Severity.Info,
+        TraceToConsole = false,
+        WriteMessagingTraces = true,
+        TraceFilePattern = @"Silo_{0}-{1}.log",
+        //TraceFilePattern = "false", // Set it to false or none to disable file tracing, effectively it sets config.Defaults.TraceFileName = null;
+        TraceLevelOverrides =
+        {
+            Tuple.Create("SiloLogStatistics", Severity.Warning),
+            Tuple.Create("Catalog", Severity.Off)
+        }
+    }
+};
+
+// Register bootstrap provider class 
+config.Globals.RegisterBootstrapProvider<AutoStartProvider>("AutoStartProvider");
+
+// Add Storage Providers
+config.Globals.RegisterStorageProvider<MemoryStorage>("MemoryStore");
+
+config.Globals.RegisterStorageProvider<AzureTableStorage>("PubSubStore",
+    new Dictionary<string, string>
+    {
+        { "DeleteStateOnClear", "true" },
+        //{ "UseJsonFormat", "true" }, // UseJsonFormat is somehow unreliable
+        { "DataConnectionString", dataConnection }
+    });
+
+config.Globals.RegisterStorageProvider<AzureTableStorage>("AzureTable",
+    new Dictionary<string, string>
+    {
+        { "DeleteStateOnClear", "true" },
+        //{ "UseJsonFormat", "true" }, // UseJsonFormat is somehow unreliable
+        { "DataConnectionString", dataConnection }
+    });
+
+config.Globals.RegisterStorageProvider<AzureTableStorage>("DataStorage",
+    new Dictionary<string, string>
+    {
+        { "DeleteStateOnClear", "true" },
+        //{ "UseJsonFormat", "true" }, // UseJsonFormat is somehow unreliable
+        { "DataConnectionString", dataConnection }
+    });
+
+config.Globals.RegisterStorageProvider<BlobStorageProvider>("BlobStorage",
+    new Dictionary<string, string>
+    {
+        { "DeleteStateOnClear", "true" },
+        { "ContainerName", "grainstate" },
+        { "DataConnectionString", dataConnection }
+    });
+
+// Add Stream Providers 
+config.Globals.RegisterStreamProvider<AzureQueueStreamProvider>("AzureQueueImplicitOnly",
+    new Dictionary<string, string>
+    {
+        { "PubSubType", "ImplicitOnly" },
+        { "DeploymentId", "orleans-stream" },
+        { "NumQueues", "4" },
+        { "GetQueueMessagesTimerPeriod", "3s" },
+        { "DataConnectionString", dataConnection }
+    });
+
+config.Globals.RegisterStreamProvider<SimpleMessageStreamProvider>("SimpleMessagingImplicitOnly",
+    new Dictionary<string, string>
+    {
+        { "PubSubType", "ImplicitOnly" }
+    });
+
+
+try
+{
+    _orleansAzureSilo = new AzureSilo();
+    var ok = _orleansAzureSilo.Start(config, config.Globals.DeploymentId, config.Globals.DataConnectionString);
+
+    //_logger.LogEnd(new { ok });
```

Comment:
this one must be deleted, it's completely unrelated to the code - we are tracking in and out of each method with these helpers.